app/src/main/java/cm/aptoide/pt/ads/MoPubInterstitialAdListener.java

```diff
@@ -0,0 +1,36 @@
+package cm.aptoide.pt.ads;
+
+import android.util.Log;
+import com.mopub.mobileads.MoPubErrorCode;
+import com.mopub.mobileads.MoPubInterstitial;
+import rx.subjects.PublishSubject;
+
+public class MoPubInterstitialAdListener implements MoPubInterstitial.InterstitialAdListener {
+
+  private PublishSubject<InterstitialClick> interstitialClick;
+
+  public MoPubInterstitialAdListener(PublishSubject<InterstitialClick> interstitialClick) {
+    this.interstitialClick = interstitialClick;
+  }
+
+  @Override public void onInterstitialLoaded(MoPubInterstitial interstitial) {
+    interstitialClick.onNext(InterstitialClick.INTERSTITIAL_LOADED);
+  }
+
+  @Override
+  public void onInterstitialFailed(MoPubInterstitial interstitial, MoPubErrorCode errorCode) {
+    Log.i("Mopub_Interstitial", errorCode.toString());
```

Comment:
Log.i does not respect the debug/production switch for logs - Also does not work with the AptoideToolbox.
We should inject Logger.getInstance() here

Reply:
Fixed